cf/samples/javascript-samples/geotab/geotab-to-sap-cloud.js

```diff
@@ -0,0 +1,768 @@
+/** 
+ * @fileoverview Code sample for demonstrating Cloud-to-Cloud integration with SAP IoT Service.
+ *
+ * The cloud service of Geotab.com is the data source. The code uses a Geotab-specific
+ * API, and the credentials for both Geotab and SAP IoT Service are necessary to run it.
+ * They must be put into the config.json file.
+ *
+ * The code is written in ECMAScript 8, which is supported by recent versions of Node.js.
+ * It polls the Geotab REST API constantly and POSTs the new data to the SAP IoT Service
+ * as soon as the data becomes available. The code is thus not supposed to run in a Web
+ * browser.
+ *
+ * Asynchronous functions (co-routines) are used intensively. Data of different types
+ * are fetched from the Geotab API and POSTed to the SAP IoT API independently of each other. 
+ * The data of a single type from a single device a posted sequentially to preserve their
+ * order and limit the load on the servers.
+ *
+ * @author Mikhail Bessonov <mikhail.bessonov@sap.com>
+ * @license SAP Sample Code License Agreement
```

Comment:
Is there any link for it?
Should we add a reference?

Reply:
Added a plain text reference to the LICENSE file, did not manage to find a link.